renovate.json5

```diff
@@ -32,7 +32,9 @@
     },
     {
       "matchFiles": ["client-next/package.json"],
-      "enabled": false
+      "matchUpdateTypes": ["patch", "minor"],
+      "groupName": "Debug UI dependencies (non-major)",
+      "schedule": ["on the first day of the week"]
```

Comment:
```suggestion
      "schedule": ["on the first day of the week"],
      "assignees" : ["testower"]
```

Reply:
I would be ok with that. In which case I would add another rule for the major upgrades, and assign them to me as well.

Reply:
Solved in [c225e1e](https://github.com/opentripplanner/OpenTripPlanner/pull/5616/commits/c225e1ec88722b25a9cb0d96386477a2f2f7870b)